docs/networking/diagnostics/inspecting-network-connections-with-netstat/index.md

```diff
@@ -2,71 +2,81 @@
 author:
   name: Mihalis Tsoukalos
   email: mihalistsoukalos@gmail.com
-description: 'A practical introduction to the netstat utility.'
-keywords: ["UNIX", "shell", "netstat", "TCP/IP", "network"]
+description: 'A practical introduction to the netstat utility on Linux, including examples of the different options available.'
+keywords: ["UNIX", "shell", "netstat", "TCP/IP", "UDP", "network", "sockets", "unix sockets", "network connections", "network statistics"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 published: 2019-09-02
 modified_by:
   name: Linode
-title: 'Inspecting network connections using netstat'
+title: 'Inspecting Network Information with netstat'
 contributor:
   name: Mihalis Tsoukalos
   link: https://www.mtsoukalos.eu/
 external_resources:
   - '[netstat](http://netstat.net/)'
 ---
 
-## Introduction
+The `netstat` command line utility shows information about the network status of a workstation or server. `netstat` is available on Unix-like and Windows operating systems, with some differences in its usage between these systems.
 
-The `netstat(8)` command line utility shows information about the network status of a UNIX machine. It is a very powerful tool that can work on the Socket, TCP, UDP, IP and Ethernet levels. Its main drawback is that it only works on the local machine. On the other hand, the main advantage of `netstat(8)` is that most of its command line options do not need root privileges to operate. Its other advantage is that it can be found on every Linux system as it is installed by default. 
+`netstat` is an older utility, and some components of its functionality have been superseded by newer tools, like the [`ss` command](/docs/networking/diagnostics/ss/). A primary benefit of using `netstat` is that it is frequently pre-installed on Linux systems, while other tools might not be. As well, many (but not all) of the command line options for `netstat` can be run without root privileges, so it can still be useful on a system where you do not have root or `sudo` privileges.
 
 {{< note >}}
-This guide is written for a non-root user. Depending on your configuration, some commands might require the help of `sudo` in order to get property executed. If you are not familiar with the `sudo` command, see the [Users and Groups](/docs/tools-reference/linux-users-and-groups/) guide.
+This guide assumes some basic knowledge of networking in Linux, including network interfaces, routing tables, and network connections and sockets.
 {{< /note >}}
 
-## Netstat Command Line Options
+## In This Guide
 
-The most important command line options of `netstat(8)` are the following:
+This guide will explore the options available when running `netstat` on Linux. `netstat` can be used to inspect:
 
-| Option | Definition |
```

Comment:
Moving the table of options down a bit. The previous flow of the guide was: 

Options table
->
Basic usage/netstat without options, plus explanation of core concepts from the output of netstat
-> 
Examples of using netstat with options

The new flow is:

Basic usage/using netstat without options, plus explanation of core concepts from the output of netstat
->
Options table
-> 
Examples of using netstat with options

Reasons for this are:
- Moves the first example further up the page, which can be beneficial because an earlier example can keep a reader engaged
- Showing the basic usage prior to the options table may help the reader understand why the options are available and useful, because they will have already seen what netstat shows by default